### ISSUE TITLE
BigQuery EXCEPT Keyword should ignore L013 "Column expression without alias. Use explicit `AS`                        | clause."

### ISSUE THREAD

**Issue by codigo-ergo-sum** [sqlfluff/sqlfluff#367]
Linting a BigQuery SQL file with a section that looks like this:

```
    title.* EXCEPT (forecast_dt,
                    id,
                    is_deleted,
                    created_ts,
                    created_by_id,
                    last_modified_ts,
                    last_modified_by_id,
                    system_modstamp_ts,
                    opp_name,
                    dbt_scd_id,
                    dbt_updated_at,
                    dbt_valid_from,
                    dbt_valid_to,
                    is_current,
                    opp_id)
```

Gives this error:
```

L:   9 | P:   5 | L013 | Column expression without alias. Use explicit `AS`
                       | clause.
```
But it doesn't really make sense and isn't possible to alias an EXCEPT statement like this, so this rule should probably not trigger rule L013.

**Comment by alanmcruickshank** [comment]
Interesting. I'll definitely take a look at this one. Are you able to provide the whole query, or at least a full select statement that includes this section which this part is found in? Some of the parser bugs depend on the context within which the rule is found so it's really useful to have a full query to test with.

**Comment by codigo-ergo-sum** [comment]
This is the same query as for issue #366 so sending the one query in an e-mail to alan@designingoverload.com so sending there. Please don't publish query directly.

**Comment by NiallRees** [comment]
I've just tested this and believe it to be fixed by #438.
